tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -278,6 +278,38 @@ def test_flatnonzero(self, xp, dtype):
         return xp.flatnonzero(array)
 
 
+@testing.parameterize(
+    {'array': numpy.random.randint(0, 2, (20,))},
+    {'array': numpy.random.randn(3, 2, 4)},
+    {'array': numpy.empty((0,))},
+    {'array': numpy.empty((0, 2))},
+    {'array': numpy.empty((0, 2, 0))},
+)
+@testing.gpu
+class TestArgwhere(unittest.TestCase):
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_equal()
+    def test_argwhere(self, xp, dtype):
+        array = xp.array(self.array, dtype=dtype)
+        return xp.argwhere(array)
+
+
+@testing.parameterize(
+    {'array': numpy.array(0)},
+    {'array': numpy.array(1)},
+)
+@testing.gpu
+@testing.with_requires('numpy>=1.17.0')
```

Comment:
Is this because the `DeprecationWarning` is issued since NumPy >= 1.17.0?

Reply:
Oh, that's my mistake. I just confirmed that NumPy does not raise any warning for `0d` on `argwhere`. it just raises that warning in the case of `nonzero`. 
But @beckernick suggested(https://github.com/cupy/cupy/pull/3135#discussion_r386174537) adding this warning to `argwhere` as well. Should we remove the warning as NumPy does not warn in that case? If not so, then I can edit the above test to check if only CuPy raises this warning or not.